src/main/java/org/kohsuke/github/GHUser.java

```diff
@@ -36,6 +35,10 @@
  */
 public class GHUser extends GHPerson {
 
+    public List<GHKey> listKeys() throws IOException {
+        return Collections.unmodifiableList(Arrays.asList(root.retrieve().to("/users/"+login+"/keys", GHKey[].class)));
```

Comment:
It is possible for there to be multiple pages of keys?  I know it is unlikely for some one to have more than a page of keys, but if it is possible it might be better to use a `PagedIterable`.

Reply:
I think it is a over kill using `PagedIterable`, but it is of course possible. Most people only have a single ssh key and less than 10. 

Personally I think it better to return a list and instead read all the keys in `listKeys()` . I can of course iterate until all key have been read

Reply:
@arngrimur-seal
I see your point.  

Okay, I just did a quick review of the code base.  Generally, `list*` methods return `PagedIterable<T>`.  If there is a method that returns a `List<T>` or a `T[]` it is called `get*`.   

I think this completely misleading naming, but I'd rather be consistent with the existing naming pattern.  So, let's change the name to `getKeys()`.  Then we're good to go.

Reply:
Not sure I understand, are we good to merge or do you want me to update the code so we  make sure all keys has been rtrieved?

Reply:
Added a suggestion comment above.